### PR TITLE
Create Add Fei as a whitelisted collateral currency.md

### PR DESCRIPTION
**Pull request by GrantGyo**

### CODE REVIEW

**Review by therealhexi — APPROVED** [review]
Looks good

File: Add-FEI-as-collateral.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+A fee of 400 was chosen as the fee for FEI, as it is the equivalent of other approved stablecoins.
+
+Implementation
+This change has no implementation other than adding the Fei token address to the collateral currency whitelist, and the fee transaction.
+
+Security Considerations
+Adding FEI as a collateral does not present any foreseeable risks to the protocol. 
+When using Fei users should account for its algorithmic nature, and four hour reweighting system.
+
+Due to FEI being an algorithmic stablecoin, it is at risk of losing the peg. However this is countered through the reweighting system, coupled with the $0.95c price floor, allowing any user to always redeem their FEI for $0.95c.
```
Comment (smb2796) [inline Add-FEI-as-collateral.md:31]: Is this $0.95 of another stable?

**Comment by smb2796** [comment]
This looks really good to me!

File: Add-FEI-as-collateral.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+UMIP Title: Add Fei as a whitelisted collateral currency
+Author: GrantG
+Status: In Discussion
+Created: 07/08/2021
+
+
+Summary (2-5 sentences)
+This UMIP will add FEI to the supported collateral currencies on the global whitelist contract, allowing the usage of FEI as collateral currency. FEI is a scalable and decentralized stablecoin that leverages protocol controlled value (PCV) for peg maintenance while maintaining highly liquid secondary markets. The PCV controlled by  the FEI DAO, gives FEI an unprecedented ability to positively affect the DeFi landscape, and currently holds roughly $500m in value.
```
Comment (chrismaree) [inline Add-FEI-as-collateral.md:8]:
```suggestion
This UMIP will add FEI to the supported collateral currencies on the global whitelist contract, allowing the usage of FEI as collateral currency. FEI is a scalable and decentralized stablecoin that leverages protocol controlled value (PCV) for peg maintenance while maintaining highly liquid secondary markets. The PCV controlled by the FEI DAO, gives FEI an unprecedented ability to positively affect the DeFi landscape, and currently holds roughly $500m in value.
```

Comment (chrismaree) [reply Add-FEI-as-collateral.md:8]: Also looks like you missed out on the headings for each section. append a `#` before Summary, Motivation ect.

File: Add-FEI-as-collateral.md (hunk 1 above)
Comment (GrantGyo) [reply Add-FEI-as-collateral.md:31]: The .95c floor is in ethereum, redeemed from the dripper.

**Comment by Reinis-FRP** [comment]
Closing this as FEI was added to Collateral Omnibus #358

**Pull request closed without merging**